app/javascript/widget/components/FooterReplyTo.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div
+    class="mb-2.5 rounded-[7px] dark:bg-slate-900 dark:text-slate-100 bg-slate-100 px-2 py-1.5 text-sm text-slate-700 flex items-center gap-2"
+  >
+    <div class="items-center flex-grow truncate">
```

Comment:
Let us use it from the Tailwind itself.

```suggestion
    <div class="items-center flex-grow overflow-hidden whitespace-nowrap text-ellipsis">
```

Reply:
https://tailwindcss.com/docs/text-overflow#truncate

`truncate` is a standard class

Reply:
Fine